rollup.config.js

```diff
@@ -135,5 +135,23 @@ export default [
       replace({'process.env.VERSION': JSON.stringify(pkg.version)}),
       sizeSnapshot(snapshotOptions)
     ]
+  },
+
+  {
+    input,
+    output: {file: pkg.unpkg, format: 'esm'},
+    plugins: [
+      nodeResolve(),
+      babel(getBabelOptions({useESModules: true})),
+      commonjs(commonjsOptions),
+      nodeGlobals({process: false}),
+      replace({
+        'process.env.NODE_ENV': JSON.stringify('development'),
+        'process.env.VERSION': JSON.stringify(pkg.version)
+      })
+      // sizeSnapshot barfs with
```

Comment:
I reported it in https://github.com/TrySound/rollup-plugin-size-snapshot/issues/24.  Should I disable `sizeSnapshot` for now?

Reply:
@TrySound is very fast usually

Reply:
I'm going to knock out the other bits of this commit.  If this is the only thing left, it might be worth disabling `size-snapshot` temporarily.  https://ajax.googleapis.com/ajax/libs/material-motion/0.1.0/material-motion.bundle.js is live now, and I'd love to get a Codepen out showing the two together this week.

Reply:
we are also in the process of releasing v10 alpha

Reply:
master branch is basically all unreleased stuff right now, I didn't know it will take so much time back then, i would have created a separate branch.

Reply:
Should we merge this with size-snapshot turned off?  Is there another branch you'd like it rebased to?

I don't mind if this is in 9 or 10 - I just want something on unpkg to point at. 🙂

Reply:
I don't know how we could do that, v9 is pre monorepo and pre rollup. The easierst way is to release v10

Reply:
we can merge with size-snapshot turned off, for sure

Reply:
Great - I think this is ready to go then.

Reply:
Just tried to rerun the build, for some reason it prints a huge amount of code, any idea where it is comming from? https://travis-ci.org/cssinjs/jss/builds/470268184

Reply:
Hmmm... I presumed that's just how Rollup works.  I don't see any obvious changes that would cause that.

Reply:
Found the culprit:
`~/Projects/jss/packages/react-jss/src/index.js → stdout...`

Now let's see if I can figure out _why_…

Reply:
Looks like I had missed `react-jss` when I added the `unpkg` keys.  Fixed now.

Reply:
oh, right, we should pack it into the starter kit as well